web/source/osk/visualKeyboard.ts

```diff
@@ -863,25 +863,6 @@ namespace com.keyman.osk {
       var tKey=this.getDefaultKeyObject();
       tKey['fontsize']=ls.fontSize;
 
-      // Identify key labels (e.g. *Shift*) that require the special OSK font
-      var specialLabel=/\*\w+\*/;
-
-      // ***Delete any empty rows at the end added by compiler bug...
-      for(n=0; n<layers.length; n++) {
-        let layer=layers[n];
-        let rows=layer['row'];
-        for(i=rows.length; i>0; i--) {
-          if(rows[i-1]['key'].length > 0) {
-            break;
-          }
-        }
-
-        if(i < rows.length) {
-          rows.splice(i-rows.length,rows.length-i);
-        }
-      }
-      // ...remove to here when compiler bug fixed ***
-
```

Comment:
Corresponds to https://github.com/keymanapp/keyman/commit/ed490fcbeffa12ecbd9dc7fdd7a42e35e5d6c720#diff-5480cfedac96cf36181f5fdcdd51aee3e44404c27c923584fceb8da14fde9b6cL43-L58